src/main/java/nl/novi/TechItEasy/controllers/TelevisionController.java

```diff
@@ -0,0 +1,116 @@
+package nl.novi.TechItEasy.controllers;
+
+import nl.novi.TechItEasy.Repositories.TelevisionRepository;
+import nl.novi.TechItEasy.exceptions.RecordNotFoundException;
+import nl.novi.TechItEasy.models.Television;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import java.util.Optional;
+
+@RestController
+@RequestMapping("/televisions")
+public class TelevisionController {
+    @Autowired
+    TelevisionRepository televisionRepository;
+
+    @GetMapping
+    public ResponseEntity<Object> getAllTelevisions() {
```

Comment:
Anders gedaan dan in de uitwerking, maar voor zover ik kan zien een nette oplossing die voldoet aan de opdracht. Zelfs met nette toevoeging van de exception wanneer de lijst leeg is (al hoeft dat volgens mij niet in geval van een "haal alles op" request.